exercises/04-iframes.md

```diff
@@ -91,13 +90,14 @@ wrapper, like this `//div[not(contains(@class,'hidden'))]/iframe`. Let's store o
 
 In most cases we might want to change iframes when we're testing. We want to be able to use our
 `Run Inside Iframe` keyword in all possible frames in our website, so we should specify the frame
-as an argument for our keyword. Furthermore, our keyword still doesn't really _do_ anything. We want
-it to be able to run _any_ keyword with _any_ arguments it might have.
+as an argument for our keyword.
 
-The keyword is farly easy to handle with a single variable. However, our issues begin when our
-keyword takes 0-n arguments and our keyword should be able to handle all situations.
+Still, our keyword still doesn't really _do_ anything yet. We want it to be able to run _any_ keyword
+with _any_ arguments it might have. If we were to pass to it one keyword without arguments, it would
+be easy. However, we want it to handle any keyword that takes 0-n arguments. Our `Run Inside Iframe`
+keyword should be able to handle all situations.
 
-In order to handle a varying amount of arguments we can use the `@{variable}` notation. Let's
+To handle a varying amount of arguments we can use the `@{variable}` notation. Let's
```

Comment:
I would add a link to User Guide for those who do not know at all what's going on with `@{variable}`s
```suggestion
To handle a varying amount of arguments we can use [the `@{variable}` notation](http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#variable-number-of-arguments-with-user-keywords). Let's
```

Reply:
Suggestion implemented.